index.html

```diff
@@ -95,6 +95,7 @@ <h3>
         </p>
         <pre class="idl">
           partial interface Navigator {
+            [SecureContext] boolean canShare(optional ShareData data);
```

Comment:
We've talked on heycam/webidl#107 about a more generic solution to feature detection of dictionary members, than every API making its own `canX` method. Perhaps we should check up on the status of that.

Reply:
Discussion is ongoing...